src/client/search/categories/CategoryFilters.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import YearSelector from './YearSelector';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+class CategoryFilters extends Component {
+  componentDidMount() {
+    $('.menu .item').tab();
+  }
+
+  componentDidUpdate() {
+    $('.menu .item').tab();
+  }
+
+  render() {
+    const currentYear = moment().year();
+    let seasonYearTabsContent = [];
+    for (let year in this.props.football.seasonsByYear) {
+      const tabContentClassName =
+        year == currentYear ? 'ui bottom attached active tab segment' : 'ui bottom attached tab segment';
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2347814857/issues/source?bid=3148782&fileBranchId=3216747#l20)